### DISCUSSION TITLE
115网盘挂载后能看到并打开目录，但是打开文件视频图片，出现重新连接1.2.3，然后提示加载失败。

### DISCUSSION THREAD

**Discussion by 4free4ever** [AlistGo/alist#3671] [Q&A] [answered]
挂载webdav后也不能读取文件。分别用电信，移动网络试过，在docker，群晖，openwrt上试过，均是此问题。。。

但是挂载在美国的vps上的一切正常。。。

是不是那个webdav策略的本地代理有问题？

**Comment by wuyifeng78** [comment, marked as answer]
将启用签名选项打开就行了。

**Reply by nyz970** [reply]
确实解决了 这个选项具体是什么作用呢

**Reply by qinguoluo** [reply]
启用签名选项在哪里？

**Reply by PreciousJyx16** [reply]
打开启用签名之后还是无法加载视频，ipad上的alistserver
